src/linear_algebra/exterior_algebra/of_alternating.lean

```diff
@@ -0,0 +1,164 @@
+/-
+Copyright (c) 2022 Eric Wieser. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Wieser
+-/
+
+import linear_algebra.clifford_algebra.fold
+import linear_algebra.exterior_algebra.basic
+
+/-!
+# Extending an alternating map to the exterior algebra
+
+## Main definitions
+
+* `exterior_algebra.lift_alternating`: construct a linear map out of the exterior algebra
+  out of alternating maps (corresponding to maps out of the exterior powers.
```

Comment:
Maybe should avoid repeating `out of` since it's being used in two different ways. How about:
```suggestion
  given alternating maps (corresponding to maps out of the exterior powers).
```